src/bvh/collision_object.hpp

```diff
@@ -215,7 +215,13 @@ namespace bvh
     ///
     /// \param[in] _data
     /// \param[in] _element_size
-    void set_entity_data_impl( const void *_data, std::size_t _element_size );
+    void set_entity_data_impl( const bvh::unmanaged_view< const std::byte * > &_data, std::size_t _element_size );
+
+    template< typename T, typename... ViewProp > auto get_bytes( Kokkos::View< const T *, ViewProp... > _data )
+    {
+      return Kokkos::View< const std::byte *, ViewProp..., Kokkos::MemoryTraits< Kokkos::Unmanaged > >(
+        reinterpret_cast< const std::byte * >( _data.data() ), _data.size() * sizeof( T ) / sizeof( std::byte ) );
+    }
```

Comment:
1. Maybe use `bvh::view`, `bvh::unmanaged_view`.
2. This does not handle reference count of the original view in any way.